common/airgapped/kustomization.yaml

```diff
@@ -1,8 +1,26 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
 kind: Kustomization
-resources:
-  - ../helm-repositories/
-patches:
-  - path: patch-add-secret-ref.yaml
-    target:
-      kind: HelmRepository
+metadata:
+  name: kommander-helm-repositories
+  namespace: kommander-flux
+spec:
+  prune: true
+  wait: true
+  interval: 6h
+  retryInterval: 1m
+  path: ../helm-repositories
```

Comment:
in case something doesn't quite work, I think I remember relative paths not working for flux kustomizations, you might have to use the root path

Reply:
thanks, updated!